warehouse/query-core/src/main/java/datawave/query/common/grouping/GroupingUtil.java

```diff
@@ -185,7 +187,15 @@ private Multimap<String,String> getFieldToFieldWithGroupingContextMap(Document d
                     // Add GroupingTypeAttribute to fieldMap with a grouping context that is based on ordered attribute type
                     int i = 0;
                     for (Map.Entry<Type<?>,GroupingTypeAttribute<?>> sortedEntry : attrSortedMap.entries()) {
-                        String fieldNameWithContext = fieldName + "." + i++;
+                        String fieldNameWithContext = null;
+                        if (containsContext) {
+                            // multivalued entries from separate entries, but the field names contain the same context number
+                            fieldNameWithContext = shortName + "." + i++;
+                        } else {
+                            // multivalued entries that do not have grouping context
+                            fieldNameWithContext = fieldName + "." + i++;
+                        }
```

Comment:
If the field contains the context, we use the shortName - Ok.

If the field doesn't contain the context, we use the fieldName.  In that case aren't fieldName, shortName, and shorterName all equal?  Why not simplify this and just use shortName all the time?

Am I missing something?